python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_adaptive_virtual_topology.schema.yml

```diff
@@ -82,6 +82,47 @@ keys:
             internet_exit_policy:
               type: str
               description: Name of the internet exit policy.
+            metric_order:
+              description: Metric order to be used for path comparison.
+              type: str
```

Comment:
it has been mentioned in the list that we should keep `metric_order` as list for future.
https://github.com/aristanetworks/avd/issues/4240#issuecomment-2257956214

please confirm once with @gmuloc @ClausHolbechArista

Reply:
We can have the metric_order as list but here is the observation:
1. If we set as list now and keep min/max length as 1 to support the current case then in future when we can have multiple order then it will be support from a specific EOS version so to support both feature we can remove the min/max length 1. But in that case the use having older EOS version need to ensure not providing more then one element in list.
2. We can keep the str now and once the multiple order supports will add one more key and will deprecate this str key same as we discussed for cipher.

Reply:
option 2 seems better, lets confirm once.

Reply:
Discussed with @ClausHolbechArista 

```
metric_order:
  type:dict
  keys:
    preferred_metric:
      type: str
      required: true (we don't want metric_order without this one)
      valid_values ...
```

Reply:
updated